app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>List of users</h1>
\ No newline at end of file
+<% @users.each do |person| %>
+  <div class="wrapper flex w80 mAuto">
+    <div>
+      <%= image_tag(person.photo, :class => 'photo border')%>
+    </div>
+    <%= link_to user_path(:id => person.id), class: 'anchor_user' do%>
+    <div class="user_info w100 border flex">
+      <h2 class="w50"><%= person.name%></h2>
+      <p class="w50 postNum">Number of posts: <%= person.postsCounter%></p>
+    </div>
+    <% end %>
+  </div>
+<% end %>
```

Comment:
- [ ] Good job so far 👏 However I would like to inform you that I was not able to view the users I have added in the database before running the server. I thought at first that this may be due to the fact that you don't have the **root route** but even after I added it, nothing was visible on screen. I highly recommend fixing your routes first so that the root route or the first page when someone visits the site will see. It's supposed to list all the users on the blog app platform. 🙏 😃 👍

Reply:
@BertrandConxy thanks for the review. This project activity doesn't ask us to give likes or add comments. Just to focus on the views.

Reply:
Here I made a loom video showing the project working. [link loom video](https://www.loom.com/share/69fbafcc1ce44d039839800af8de4c07)

Reply:
@BertrandConxy I already push a commit with the root route working.